### ISSUE TITLE
Can this method obtain absolute depth?

### ISSUE THREAD

**Issue by kowloontong** [LinShan-Bin/OccNeRF#4]
Hello, thank you for your great work, I was wondering whether this self-supervised method can obtain absolute depth.

**Comment by LinShan-Bin** [comment]
Yes. Since we use the ground truth poses, the predicted depth is naturally absolute depth.
